readme.md

```diff
@@ -1,22 +1,24 @@
-# Jumpstart Live (JSL)
-_In person program for [Ada Developers Academy](http://adadevelopersacademy.org/) students who are not yet confident using loops, arrays, and hashes_
+# JumpStart Live (JSL)
+_In person program for [Ada Developers Academy](http://adadevelopersacademy.org/) students who are not yet well-versed using loops, arrays, and hashes_
```

Comment:
just say .. for _some_ students or similar and drop the well-versed please

Reply:
Done.